tox.ini

```diff
@@ -24,15 +24,15 @@ deps =
     dask[array,diagnostics]
     setuptools_scm
 commands =
-    pytest --cov=unyt --cov-append --doctest-modules --doctest-plus --doctest-rst --basetemp={envtmpdir} -W once
+    pytest --cov=unyt --cov-append --doctest-modules --doctest-plus --doctest-rst --basetemp={envtmpdir}
```

Comment:
Can you explain a little more why you made this change?

Reply:
I found it was necessary in order to get all deprecation warnings poping up in doctests instead of just the first one calling `warn_deprecated`